### ISSUE TITLE
As a user, I want to be able to reset my password if I forget it.

### ISSUE THREAD

**Issue by KatherineGoldklang** [innovationprogrammeuki/Absences#39]
AC01: Can I reset my password from the login screen if I forget it?

**Comment by natalietolley** [comment]
![forgot password 001](https://cloud.githubusercontent.com/assets/20187071/19191658/4e0584d6-8c9b-11e6-92cc-dc963276f1a3.jpeg)

**Comment by JohnThirlaway1** [comment]
Linking this functionality with story 37, there needs to be a meeting regarding the best way of generating the email and sending it to the user, wether it will be using a 3rd party API to handle the response or using mailto to trigger the admins client and launch an already formatted email template.

**Comment by stevenhaynes** [comment]
If your blocked on this can you pick up another story in the meantime and we'll look at this later?

**Comment by JohnThirlaway1** [comment]
yes I will pick up another story in the meantime
